plugins/figma/manifest.json

```diff
@@ -8,5 +8,11 @@
   "enableProposedApi": false,
   "editorType": ["figma"],
   "documentAccess": "dynamic-page",
-  "networkAccess": { "allowedDomains": ["https://api.github.com"] }
+  "networkAccess": {
+    "allowedDomains": [
+      "https://api.github.com",
+      "https://fonts.googleapis.com",
```

Comment:
What do we need this for?

Reply:
Figma was complaining about it. It seems like google fonts is used in `App.css`

Reply:
Suggest we change it to use our own from altinncdn as we recommend our users to do.